docker/config/my.env.dist

```diff
@@ -9,3 +9,6 @@
 #
 # Set this to the Mapbox API token.
 # MAPBOX_TOKEN=
+
+# Set this if you're having serious problems that need Pyramid debugging.
+# PYRAMID_DEBUG_ALL=True
```

Comment:
Figured I'd add this. It shows template data and some other things. It's probably interesting for certain scenarios.